internal/sysfs/futimens.go

```diff
@@ -50,16 +50,11 @@ func Utimens(path string, times *[2]syscall.Timespec, symlinkFollow bool) syscal
 	return platform.UnwrapOSError(err)
 }
 
-var _zero uintptr //nolint:unused
```

Comment:
thanks @evacchi for figuring this out. Futimes tests failed on 1.21 until this was replaced with nil